games/space-invaders/Space Invaders [David Winter].json

```diff
@@ -0,0 +1,108 @@
+{
+  "title": "Space Invaders",
+  "authors": [
+    {
+      "id": "david-winter",
+      "name": "David Winter"
+    }
+  ],
+  "release": "1978",
+  "category": "GAME",
+  "description": "The well known game. Destroy the invaders with your ship.",
```

Comment:
Description should not contain information about key bindings. It duplicates the information under `keys`.

Reply:
Agreed, but this description was copied from "somewhere" (David Winter's website or documentation in his games bundle, I don't recall). I haven't rewritten anything in here from its original wording, but there's no reason we couldn't do that of course.

Reply:
Yes, it was copied. Maybe we could provide `originalDescription` or `authorsNote` for completeness. What do you think?